src/main/sql/schema-postgresql.sql

```diff
@@ -0,0 +1,76 @@
+CREATE TABLE BATCH_JOB_INSTANCE  (
```

Comment:
Isn't this and the schema file already included in the framework's JAR files?

Reply:
I copied from the Spring Academy course here too.

This can likely be deleted from my repo as I'm not using postgres anymore and these tables are created automatically when using H2.